.github/workflows/apply.sh

```
@@ -0,0 +1,130 @@
+
+name=$(user_name "$LOGIN")
+email=$(user_email "$LOGIN" "$name")
+git config set user.name "$name"
+git config set user.email "$email"
+
+git remote add pr $PR_HEAD_URL
+git fetch pr
+git checkout -t pr/$PR_HEAD_REF
+
+tmp=$(mktemp -d)
+trap "rm -rf -- $tmp" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)